AdminApi/src/AdminApi/Controllers/IdentitiesController.cs

```diff
@@ -112,6 +113,15 @@ public async Task<IActionResult> StartDeletionProcessAsSupport([FromRoute] strin
         var response = await _mediator.Send(new StartDeletionProcessAsSupportCommand(address), cancellationToken);
         return Created("", response);
     }
+
+    [HttpGet("{address}/DeletionProcess/{id}")]
+    [ProducesResponseType(typeof(GetDeletionProcessAsSupportQuery), StatusCodes.Status200OK)]
+    [ProducesError(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> GetDeletionProcessAsSupport([FromRoute] string address, [FromRoute] string id, CancellationToken cancellationToken)
```

Comment:
I think the `address` should be renamed to `identityAddress` and `id` to `deletionProcessId` or similar. Naming of similar parameters in other methods is not consistent though, so this is merely a suggestion.